specification/datalake-analytics/data-plane/readme.nodejs.md

```diff
@@ -7,9 +7,22 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 nodejs:
   azure-arm: true
   package-name: azure-arm-datalake-analytics
+  package-version: 3.0.0-preview
+  output-folder: $(node-sdks-folder)/lib/services/dataLake.Analytics
+  generate-license-txt: false
+  generate-package-json: false
+  generate-readme-md: false
   batch:
-    - output-folder: $(node-sdks-folder)/lib/services/dataLake.Analytics/lib/catalog
-      tag: package-catalog
-    - output-folder: $(node-sdks-folder)/lib/services/dataLake.Analytics/lib/job
-      tag: package-job
+    - tag: package-catalog
+    - tag: package-job
 ```
+
+``` yaml $(nodejs) && $(tag) == 'package-catalog'
```

Comment:
Beyond the output-folder changing to source-code-folder-path, is this a style change or a correctness change to add the additional yaml blocks?

Reply:
@lmazuel showed me to do it this way when I configured the Java Swagger To SDK readme files and I feel like it's a little easier to read (style change). I'm not sure if one way is more correct than the other.

Reply:
In this particular scenario, I feel both are fine.
However, for multi-api versionning, we want to have ONE "batch" at the beginning of the file to explain what ApiVersions are shipped by all SDKs. Then, this batch will be just a list of tags.